apps/web/app/api/cron/weekly-summary/email.tsx

```diff
@@ -0,0 +1,94 @@
+import { CreateReminderNotificationBody } from "@/app/api/cron/weekly-summary/components/CreateReminderNotificationBody";
+import { LiveSurveyNotification } from "@/app/api/cron/weekly-summary/components/LiveSurveyNotification";
+import { NotificationFooter } from "@/app/api/cron/weekly-summary/components/NotificationFooter";
+import { NotificationHeader } from "@/app/api/cron/weekly-summary/components/NotificationHeader";
+import { NotificationInsight } from "@/app/api/cron/weekly-summary/components/NotificationInsight";
+import { render } from "@react-email/render";
+
+import { WEBAPP_URL } from "@formbricks/lib/constants";
+import { EmailTemplate } from "@formbricks/lib/emails/EmailTemplate";
+import { sendEmail } from "@formbricks/lib/emails/emails";
+
+import { TNotificationResponse } from "./types";
+
+const getEmailSubject = (productName: string): string => {
+  return `${productName} User Insights - Last Week by Formbricks`;
+};
+
+export const sendWeeklySummaryNotificationEmail = async (
+  email: string,
+  notificationData: TNotificationResponse
+) => {
+  const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+
```

Comment:
can we declare this array outside of this function, so that it can be reused in the function below?